arpav_ppcv/config.py

```diff
@@ -160,6 +160,7 @@ class ArpavPpcvSettings(BaseSettings):  # noqa
     templates_dir: Optional[Path] = Path(__file__).parent / "webapp/templates"
     static_dir: Optional[Path] = Path(__file__).parent / "webapp/static"
     thredds_server: ThreddsServerSettings = ThreddsServerSettings()
+    nearest_station_radius_meters: int = 10_000
```

Comment:
good find - I had forgotten about that - I'll change it

Reply:
Ouch, nevermind my previous comment. This is already available as an env variable - all settings keys are.

It can be provided by setting `ARPAV_PPCV__NEAREST_STATION_RADIUS_METERS=<some-int>` as an env var.